lobby.go

```diff
@@ -207,6 +207,50 @@ func wsListen(lobby *Lobby, player *Player, socket *websocket.Conn) {
 						lobby.Drawer.WriteAsJSON(JSEvent{Type: "your-turn"})
 					}
 				}
+			} else if received.Type == "on-kick-button" {
+				toKickID, isString := (received.Data).(string)
+				toKick := -1
+				if !isString {
+					fmt.Println("Invalid data")
+					continue
+				}
+
+				for index, otherPlayer := range lobby.Players {
+					if otherPlayer.ID == toKickID {
+						toKick = index
+					}
+				}
+
+				if toKick != -1 {
+					if player.votedForKick[toKickID] {
+						continue
+					}
+
+					player.votedForKick[toKickID] = true
+					playerToKick := lobby.Players[toKick]
+					playerToKick.voteKickCount++
+
+					if playerToKick.voteKickCount > 2 {
```

Comment:
We might want to make this somewhat dependent on the amount of players. Like what if you have 24 people and two of them could kick everyone?